src/Html/Renderables/Element.php

```diff
@@ -0,0 +1,263 @@
+<?php
+
+namespace Erusev\Parsedown\Html\Renderables;
+
+use Erusev\Parsedown\Html\Renderable;
+use Erusev\Parsedown\Html\Sanitisation\CharacterFilter;
+use Erusev\Parsedown\Html\Sanitisation\Escaper;
+
+final class Element implements Renderable
+{
+    use CanonicalStateRenderable;
+
+    /** @var array<string, true> */
+    public static $TEXT_LEVEL_ELEMENTS = [
+        'a' => true,
+        'b' => true,
+        'i' => true,
+        'q' => true,
+        's' => true,
+        'u' => true,
+
+        'br' => true,
+        'em' => true,
+        'rp' => true,
+        'rt' => true,
+        'tt' => true,
+        'xm' => true,
+
+        'bdo' => true,
+        'big' => true,
+        'del' => true,
+        'img' => true,
+        'ins' => true,
+        'kbd' => true,
+        'sub' => true,
+        'sup' => true,
+        'var' => true,
+        'wbr' => true,
+
+        'abbr' => true,
+        'cite' => true,
+        'code' => true,
+        'font' => true,
+        'mark' => true,
+        'nobr' => true,
+        'ruby' => true,
+        'span' => true,
+        'time' => true,
+
+        'blink' => true,
+        'small' => true,
+
+        'nextid' => true,
+        'spacer' => true,
+        'strike' => true,
+        'strong' => true,
+
+        'acronym' => true,
+        'listing' => true,
+        'marquee' => true,
+
+        'basefont' => true,
+    ];
+
+    /** @var string[] */
+    public static $COMMON_SCHEMES = [
+        'http://',
+        'https://',
+        'ftp://',
+        'ftps://',
+        'mailto:',
+        'tel:',
+        'data:image/png;base64,',
+        'data:image/gif;base64,',
+        'data:image/jpeg;base64,',
+        'irc:',
+        'ircs:',
+        'git:',
+        'ssh:',
+        'news:',
+        'steam:',
+    ];
+
+    /** @var string */
+    private $name;
+
+    /** @var array<string, string>*/
+    private $attributes;
+
+    /** @var Renderable[]|null */
+    private $Contents;
+
+    /**
+     * @param string $name
+     * @param array<string, string> $attributes
+     * @param Renderable[]|null $Contents
+     */
+    public function __construct($name, $attributes, $Contents)
+    {
+        $this->name = $name;
+        $this->attributes = $attributes;
+        $this->Contents = $Contents;
```

Comment:
Why is it Capitalized?

Reply:
Bit of a personal convention, but reminds me that I'm dealing with an object and not a scalar since these have notably different behaviours aside from their basic interface (e.g. reference v.s value types, whether you want `===` to mean strictly equal or identity, whether they have a possibility of being falsey...)